DigitalLearningSolutions.Data/Services/ActivityService.cs

```diff
@@ -126,6 +132,28 @@ public byte[] GetActivityDataFileForCentre(int centreId, ActivityFilterData filt
             return stream.ToArray();
         }
 
+        public (DateTime startDate, DateTime? endDate)? GetValidatedUsageStatsDateRange(
+            string startDateString,
+            string endDateString,
+            int centreId
+        )
+        {
+            var parsedStartDate = DateTime.Parse(startDateString);

```

Comment:
Could this just be startDate? Also what happens if parsing fails - we should just handle it as not found rather than throwing